server/src/main/java/com/google/coffeehouse/common/StorageHandler.java

```diff
@@ -102,61 +224,110 @@ public static String getClubQuery(DatabaseClient dbClient, String clubId) {
     Statement statement = 
         Statement.newBuilder(
-                "SELECT bookId, author, isbn, title "
-                  + "FROM Books "
-                  + "WHERE bookId = @bookId")
-            .bind("bookId")
-            .to(bookId)
+                "SELECT userId, clubId, membershipType "
+                  + "FROM Memberships "
+                  + "WHERE clubId = @clubId AND membershipType = " + Membership.MEMBER)
+            .bind("clubId")
+            .to(clubId)
             .build();
-    if (resultCount == 1) {
+    if (resultCount > 0) {
       try (ResultSet resultSet = dbClient.singleUse().executeQuery(statement)) {
         while (resultSet.next()) {
-          title = resultSet.getString("title");
-          author = (resultSet.isNull("author") || resultSet.getString("author").isEmpty())
-              ? NO_AUTHOR : "Author: " + resultSet.getString("author");
-          isbn = (resultSet.isNull("isbn") || resultSet.getString("isbn").isEmpty())
-              ? NO_ISBN : "ISBN: " + resultSet.getString("isbn");
+          String userId = resultSet.getString("userId");
+          persons.add(getPersonQuery(dbClient, userId));
         }
-        bookInfo = String.format(
-                          "Book ID: %s || %s || %s || Title: %s\n",
-                          bookId, author, isbn, title);
       }
-    } else if (resultCount > 1) {
-      bookInfo = ERROR_MORE_THAN_ONE_BOOK;
     } else {
-      bookInfo = BOOK_DOES_NOT_EXIST;
+      throw new IllegalArgumentException(Membership.NO_MEMBERS);
     }
-    return bookInfo;
+    return persons;
+  }
+
+  /**
+  * Creates and returns a list of {@link Club}s depending on the user's membership status.
+  * This method builds a {@link Club} with a club's ID, book ID, description, name, and ownerID
+  * for each club that a user is either a member of or not a member of, and appends each club
+  * to a list of clubs that gets returned.
+  * @param  dbClient    the database client
+  * @param  userId      the user ID string used to query and get a list of clubs
+  * @return clubs       the list of Club objects
+  */
+  public static List<Club> getListOfClubsQuery(DatabaseClient dbClient, String userId, Membership.MembershipStatus membershipStatus) {
+    List<Club> clubs = new ArrayList<>();
+    if (membershipStatus == Membership.MembershipStatus.MEMBER) {
+      Statement statement = 
+        Statement.newBuilder(
+                "SELECT userId, clubId, membershipType "
+                  + "FROM Memberships "
+                  + "WHERE userId = @userId AND "
+                  + "membershipType = " + Membership.MEMBER)
+            .bind("userId")
+            .to(userId)
+            .build();
+      try (ResultSet resultSet = dbClient.singleUse().executeQuery(statement)) {
+        while (resultSet.next()) {
+          String clubId = resultSet.getString("clubId");
+          clubs.add(getClubQuery(dbClient, clubId));
+        }
```

Comment:
Duplicated on lines 317 -> 321. This can easily be avoided by declaring `Statement statment`; outside of the if block and then running this code at the end of the function.